### ISSUE TITLE
How to force Microsoft Office Addin to use Edge Webview2 runtime?

### ISSUE THREAD

**Issue by jfoclpf** [OfficeDev/office-js#2176]
I read this nice [article](https://devblogs.microsoft.com/microsoft365dev/understanding-office-add-ins-runtime/) from Microsoft explaining how the Addin runtime works.

I am using **Windows 10 Enterprise** and the system has already **Microsoft Edge WebView2**: 

[![enter image description here][1]][1]

My Office 365 version is **16.0.13801.20442 32-bit** and hence it is Microsoft 365 ver. >= 16.0.13530.20424, which according to [this table](https://docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins) would force the Addin to use Microsoft Edge with WebView2. 

[![enter image description here][2]][2]

But it is not using it. It is using the legacy EdgeHTML runtime

[![enter image description here][3]][3]

## TLDR 

If these 2 conditions are met:

 - Microsoft 365 ver. >= 16.0.13530.20424
 - Edge WebView2 installed

Why the runtime of Addin is the old EdgeHTML?

  [1]: https://i.stack.imgur.com/QxhyF.png
  [2]: https://i.stack.imgur.com/qGweX.png
  [3]: https://i.stack.imgur.com/X5ljI.png

**Comment by Dengshan2018** [comment]
@jfoclpf there are a few cases that the runtime will fall back to legacy webview. could you check:
1. if your IT is blocking enable webview2 by checking if all machines are running on legacy webview.
2. your machine is WIP enabled. when WIP enabled, runtime will fall back to legacy webview. webview2 doesn't support WIP yet.

**Comment by jfoclpf** [comment]
it's solved, https://stackoverflow.com/a/69575012/1243247
